lib/emque/producing/publisher/rabbitmq.rb

```diff
@@ -5,6 +5,9 @@ module Emque
   module Producing
     module Publisher
       class RabbitMq < Emque::Producing::Publisher::Base
+        Emque::Producing::Message::PUBLISH_IGNORED_EXCEPTIONS << Bunny::Exception
+        Emque::Producing::Message::PUBLISH_IGNORED_EXCEPTIONS << Timeout::Error
```

Comment:
Is this a circular dependency? It feels a little awkward to reach into the Message class from here.  Did the idea of throwing emque exceptions from this class to be caught by the Message class not work well?